frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -56,6 +56,7 @@ export function MissionHeader({ mission }: MissionHeaderProps) {
             <StatusReason mission={mission}></StatusReason>
             <InfoSection>
                 <MissionStatusDisplay status={mission.status} />
+                {HeaderText(TranslateText('Date') + ': ' + startDate)}
```

Comment:
Needs to add Date to the "no.json" and "en.json" for a translation to exist

Reply:
It is already there :)